File: python/gpt/p17.py
# p17: take a number from user and return value at that index or -1 if out of bounds


def lookup(idx):
    arr = [5, 10, 15, 20, 25]
    if idx < 0 or idx >= len(arr):
        return -1
    return arr[idx]


def main():
    try:
        idx = int(input("Enter index: "))
    except Exception:
        print(-1)
        return 0
    print(lookup(idx))
    return 0


if __name__ == "__main__":
    main()

File: javascript/gpt/p17.js
// p17: take number from user and return value at that index or -1
const readline = require("readline");

function lookup(idx) {
  const arr = [5, 10, 15, 20, 25];
  if (idx < 0 || idx >= arr.length) return -1;
  return arr[idx];
}

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});
rl.question("Enter index: ", (ans) => {
  const idx = parseInt(ans.trim(), 10);
  if (Number.isNaN(idx)) console.log(-1);
  else console.log(lookup(idx));
  rl.close();
});
